Makefile

```diff
@@ -22,6 +23,12 @@ build-backend: ## Build backend only
 build-cluster-agent: ## Build cluster-agent only
 	cd $(MAKEFILE_ROOT)/cluster-agent && make build
 
+docker-build: ## Build docker image
+	docker build -t ${IMG} $(MAKEFILE_ROOT)
+
+docker-push: ## Push docker image - note: you can change the name with 'docker push IMG=quay.io/USERNAME/gitops-service:v0.0.1'
+	docker push ${IMG}
```

Comment:
So, This is more a question than a suggestion, can't we add tags var as well here to point something like:
`docker push $(IMG):$(IMAGE_TAG)` So in future if we have various tags along with latest, then we can just use that functionality?

Reply:
I am little bit confused. You can do this:

```
NAME="gitops-service"
TAG="v0.0.1"
make docker-push IMG="$NAME:$TAG"
```

what am I missing?

Reply:
This also works, I was just suggesting to have those $NAME and $TAG defined individually instead of just $IMG. But yes, this works absolutely! :smile_cat: